OpenDreamClient/Interface/DreamInterfaceManager.cs

```diff
@@ -302,7 +302,7 @@ public void FrameUpdate(FrameEventArgs frameEventArgs) {
             DefaultMap.Viewport.Eye = _eyeManager.CurrentEye;
     }
 
-    public InterfaceElement? FindElementWithId(string id) {
+    public InterfaceElement? FindElementWithId(string id, bool selectDefault = false) {
```

Comment:
There are a lot of calls to `FindElementWithId()` that would need to explicitly set `selectDefault`. I think it's better to instead have the colon be a part of `id` so it works more universally.